atoma-proxy-service/src/proxy_service.rs

```diff
@@ -498,7 +592,7 @@ async fn get_all_subscriptions(
     )
```

Comment:
Missing path argument `task_id` in utoipa docs, [see this for reference ](https://github.com/atoma-network/atoma-node/blob/main/atoma-daemon/src/handlers/almost_filled_stacks.rs#L43C1-L45C7)

Reply:
Great ! Thanks for spotting this